oaknational/Oak-Web-Application
# feat: get social-sharing-image from sanity

## Description

- get social-sharing-image from sanity
- removes old file from codebase
- removes redundant `generate_brand_assets` script
- removes social sharing image from config getters

**Todo: after PR goes in, remove sharing image from config json**

## Issue(s)

Progresses #1388 

## How to test

1. Go to {cloud link}
2. Check the meta image tags, download the linked image
3. Should be same as it is now (with same url) 
![image](https://user-images.githubusercontent.com/12934669/231740507-4c4b9f50-f1f4-4bbf-a061-59b80aef97b6.png)
